pkg/controller/ambassadorinstallation/reconcile.go

```diff
@@ -376,3 +418,11 @@ func (r *ReconcileAmbassadorInstallation) updateResourceStatus(o *unstructured.U
 	o.Object["status"] = status
 	return r.Client.Status().Update(context.TODO(), o)
 }
+
+// Report sends an event to Metriton
+func (r *ReconcileAmbassadorInstallation) Report(eventName string, meta ...ScoutMeta) {
```

Comment:
I seem to me every time the `r.Report()` function is called, the `eventName` parameter is more-or-less of a log statement. I think we should standardize all the reported `eventName` to a fixed keyword (`start_reconcile`, `previous_install`, etc.) so we can extract trace reports based on expected event names rather than fuzzy log strings.

Reply:
That is a good suggestion, thanks.

Reply:
would you then report errors in the metadata?  e.g. `r.Report("start_reconcile", ScoutMeta("Error", message)`?